cli/ml_training.go

```diff
@@ -0,0 +1,114 @@
+package cli
+
+import (
+	"github.com/urfave/cli/v2"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+// MLTrainingUploadAction retrieves the logs for a specific build step.
+func MLTrainingUploadAction(c *cli.Context) error {
+	client, err := newViamClient(c)
+	if err != nil {
+		return err
+	}
+
+	metadata := createMetadata(c.Bool(mlTrainingFlagDraft), c.String(mlTrainingFlagType),
+		c.String(mlTrainingFlagFramework))
+	metadataStruct, err := convertMetadataToStruct(metadata)
+	if err != nil {
+		return err
+	}
+
+	if _, err := client.uploadPackage(c.String(generalFlagOrgID),
```

Comment:
It doesn't seem like we do any validation at all. Maybe this is out of scope but couldn't I upload any random file this way (and publish in the registry)?

Reply:
Great point, that is true yes. Validation would be a part of this ticket: https://viam.atlassian.net/browse/DATA-2487. Once the validation in the BE is in place (as part of the package upload implementation), we would want the CLI to also perform those validations before upload.